pyfixest/estimation/fepois_.py

```diff
@@ -387,32 +387,20 @@ def predict(
             Another stopping tolerance for scipy.sparse.linalg.lsqr().
             See https://docs.scipy.org/doc/scipy/reference/generated/scipy.sparse.linalg.lsqr.html
 
-
-
         Returns
         -------
         np.ndarray
             A flat array with the predicted values of the regression model.
         """
-        _Xbeta = self._Xbeta.flatten()
-        _has_fixef = self._has_fixef
-
-        if _has_fixef:
+        if self._has_fixef:
             raise NotImplementedError(
                 "Prediction with fixed effects is not yet implemented for Poisson regression."
             )
         if newdata is not None:
             raise NotImplementedError(
                 "Prediction with function argument `newdata` is not yet implemented for Poisson regression."
             )
-
-        # y_hat = super().predict(newdata=newdata, type=type, atol=atol, btol=btol)
```

Comment:
Where in the new version are we conducting the exponentiation for for the Poisson prediction for type = "response"?

Reply:
Ah, I see, we do it in the fit method already.